### PR TITLE
 DEP-147: Add owasp-java-html-sanitizer library to encode data in pre…

### PR DESCRIPTION
**Pull request by kmenzli**
…sentation layer

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -131,6 +131,7 @@
     <org.mockito.version>1.10.19</org.mockito.version>
     <org.ogce.xpp3.version>1.1.6</org.ogce.xpp3.version>
     <org.owasp.antisamy.version>1.5.3</org.owasp.antisamy.version>
+    <com.googlecode.owasp-java-html-sanitizer.owasp-java-html-sanitizer.version>20160413.1</com.googlecode.owasp-java-html-sanitizer.owasp-java-html-sanitizer.version>
```
Comment (thomasdelhomenie) [inline pom.xml:134]: "owasp-java-html-sanitizer" is duplicated in the property name.

File: pom.xml [hunk 2]
```diff
@@ -740,6 +741,11 @@
         <version>${org.owasp.antisamy.version}</version>
       </dependency>
       <dependency>
+        <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
```
Comment (thomasdelhomenie) [inline pom.xml:744]: The dependency list is sorted by alphabetical order of the groupId ( @mgreau ;-) )

File: pom.xml (hunk 1 above)
Comment (kmenzli) [inline pom.xml:134]: I used groupId:artifactId

File: pom.xml (hunk 2 above)
Comment (kmenzli) [inline pom.xml:744]: Before release @mgreau uses a maven plugin to sort dep, isn't it?

**Comment by kmenzli** [comment]
I'll provide a new one

**Pull request closed without merging**